python/ray/data/_internal/execution/backpressure_policy/streaming_output_backpressure_policy.py

```diff
@@ -39,6 +42,8 @@ class StreamingOutputBackpressurePolicy(BackpressurePolicy):
         "backpressure_policies.streaming_output.max_blocks_in_op_output_queue"
     )
 
+    OUTPUT_DETECTION_INTERVAL_SECONDS = 1
```

Comment:
This means allow outputting 1 block per second. It's already very conservative. 
Even if the first block is slow to come, the amount of unnecessarily output data is still small.
I also tested https://github.com/ray-project/ray/issues/41496. it can finish under the similar time with 1s, while 10s will slow it down by 10x.